init.sh

```diff
@@ -43,7 +43,10 @@ changetype: delete
 
 EOF
 
-ldapmodify -x -D "cn=admin,${SUFFIX}" -w "${LDAP_ADMIN_PASSWORD}" -f /tmp/delete.ldif
+if [ ! -z "${LDAP_INI_GROUP_1}" ]; then
```

Comment:
As I said in Slack, I think we should execute the `delete.ldif` always in order to make sure that no `dc=dotdata,dc=com` and `cn=admin,dc=dotdata,dc=com` entries do not exist before the replication happens even in `sa_openldap_replica`.